packages/react-components/react-drawer/src/e2e/DrawerShared.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import { mount } from '@cypress/react';
+import { FluentProvider } from '@fluentui/react-provider';
+import { webLightTheme } from '@fluentui/react-theme';
+import { DrawerProps } from '@fluentui/react-drawer';
+
+const mountFluent = (element: JSX.Element) => {
+  mount(<FluentProvider theme={webLightTheme}>{element}</FluentProvider>);
+};
+
+export function testDrawerBaseScenarios(Component: React.ComponentType<DrawerProps>) {
```

Comment:
The shared tests in a function that can be called then for each base Drawer component.